templates/includes/utils.html.twig

```diff
@@ -8,3 +8,6 @@
     {% endif %}
 {% endmacro %}
 
+{% macro uniqueId() -%}
+    {{ 'now'|date('YmdHisu') }}
```

Comment:
Can you please create a twig function `tabler_unique_id()` for that, the current approach looks dangerous.

Reply:
At the beginning, I was about to add a twig function, but I thought it was overpower for a template Bundle. But I'm good with that!

> approach looks dangerous

Why ? I'll add uniqueId() PHP function that do the same as the current twig function : `Gets a prefixed unique identifier based on the current time in microseconds`.
Just tell me why you find that dangerous not why I should do it (I'll do it anyway 😉)

Reply:
> I am getting old

😆😆😆😆

Funny but I have the same thought on default UUID value from SQL. 
Same as uniqId it is generated by `microseconds` and others.. I was frustrated than generating multiple UUID from a big INSERT will create duplicates.
 
ATE, microseconds ids are 99% non reproducible, unless you change the server time 👀

Reply:
Oh, good to know. 
I thought UUID is truly unique ... shit, I am using it in one project as well 😨 
The server time-change is happening twice a year during summer-winter shift, so that is really dangerous for database (not twig though).

Reply:
You have 1/10 000 000 chance to generate the same UUID V1 for ONE same second.
https://en.wikipedia.org/wiki/Universally_unique_identifier#Versions

More chance than winning the EuroMillion, but still slight 😉

Reply:
Ah well then, I guess I am fine to live with that chance